src/settings/utils.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const NAME_REGEXP = new RegExp(/[^a-zA-Z\d]/);
```

Comment:
I don't know exact requirement, but it limits values only for engl letters

just curious

Reply:
Validation should be the same as on [BE side](https://github.com/folio-org/acq-models/blob/master/mod-orders-storage/schemas/purchase_order.json#L60)

Reply:
IIUC, by link you provided there is a length check as well. So it's not the same validation as on back-end.

Reply:
Right. But BE length validation covers all `poNumber` with `prefix` and `suffix`. There are no requirements for length limits for prefix or suffix. I mean it's possible to use `prefix`(11 chars)+`poNumber`(5 chars) with no `suffix`. It's a question to PO about prefix/suffix length limitations

Reply:
so back-end work this any length of suffix/prefix?

Reply:
There are no limits on the BE side for suffix/prefix length in settings. But users can face length validation during PO creation.

Reply:
it's allowed to use number in `poNumber`, does it mean su/prefix can contain numbers as well?

Reply:
ah I missed `d`, ignore previous one